src/displayUI.js

```diff
@@ -18,15 +19,16 @@ export function displayOneCategory(categoryElement) {
     instanceOfMainClass.markAsImportant(categoryElement);
   })
 
-  categoryItem.addEventListener("click", (e) => {
+  category.addEventListener("click", (e) => {
     e.preventDefault();
     document.querySelector("#oneCategoryDisplayBox").innerHTML = "";
     let firstWord = categoryElement.name;
     let parameter = firstWord.split(" ")[0];
-    instanceOfMainClass.displayLinks(parameter);
+    instanceOfMainClass.collectAllLinks(parameter);
     instanceOfMainClass.setTextFilter(parameter);
+    selectedCategoryDisplay(firstWord);
```

Comment:
What is purpose of selectedCategoryDisplay?
When something selected is enough style/class change.

Reply:
I tought you said that beside it is highlited, is should be visible under all categories as well, just like on that car site that we saw.

Reply:
Ok but every UI state, should be covered in model, so we need to have information what is selected category so to we know our state, and we could rerender ui.